RallyHereDebugTool/Source/Private/RHDTW_Entitlements.cpp

```diff
@@ -167,6 +189,113 @@ void FRHDTW_Entitlements::Do()
 				ImGui::EndTable();
 			}
 		}
-		ImGui::PopID();
```

Comment:
I don't see a corresponding PushID removed, is this okay?

Reply:
The push Id was commented out above, and this was causing layout problems because it was popping more than was being pushed.